results/voyageai__voyage-3-lite/no_revision_available/model_meta.json

```diff
@@ -0,0 +1 @@
+{"name": "voyageai__voyage-3-lite", "revision": "no_revision_available", "release_date": "2024-09-22", "languages": null, "n_parameters": null, "memory_usage": null, "max_tokens": 32000, "embed_dim": 512, "license": null, "open_source": false, "similarity_fn_name": null, "framework": [], "loader": "VoyageWrapper"}
```

Comment:
Does it use the MTEB voyage wrapper?

Reply:
(if so I believe we can merge this in - otherwise we can simply leave it empty)

Reply:
Yes confirmed all results come from `mteb/models/voyage_models.py`.